src/shared/components/oncoprint/controls/OncoprintControls.tsx

```diff
@@ -1638,8 +1280,7 @@ export default class OncoprintControls extends React.Component<
         return (
             <div className="oncoprint__controls">
                 <ButtonGroup>
-                    <this.ClinicalTracksMenu />
-                    <this.HeatmapMenu />
+                    <this.AddTracksMenu />
```

Comment:
this is kinda weird.  like, why not just ```{ this.AddTracksMuenu }```.   @adamabeshouse is there a reason?

Reply:
this lets `AddTracksMenu` be an observer component, rather than a computed. I think this gives benefits in compartmentalizing which observables cause changes in various parts of the UI